[SYNTH-11931] Create README for the integration (#6)
DataDog/synthetics-test-automation-bitrise-step-upload-application: README.md

# Datadog Continuous Testing for Bitrise

<!-- TODO add link to marketplace after we publish the step -->
<!-- [![Visual Studio Marketplace Version]()][1001]  -->

> [![Build Status](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4/status.svg?token=480MdFpG78E6kZASg5w1dw&branch=main)](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4)

Reviewer: I noticed the link to the build status was wrong in the other PR, but correct here 🤔 

https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/pull/12

Author: It was correct but actually pointing to the wrong place 😅

[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)

# Overview

> With the `synthetics-test-automation-bitrise-step-upload-application` step you can upload a new version of your application to Datadog to run Synthetics tests against during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2002]. Your application already needs to exist for this to work.

Reviewer 2: ```suggestion
With the `synthetics-test-automation-bitrise-step-upload-application` step, you can upload a new version of your application to Datadog to run Synthetics tests against during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2002]. Your application already needs to exist for this to work.
```

## Setup

To get started:

1. Add this step to your workflow. You can see [Bitrise's documentation][4001] on how to do that. You can also configure it locally by referencing this step in your `bitrise.yml`.

> 2. Add your API and App Keys to your secrets in Bitrise. Documentation on how to do that can be found here - [Setting a Secret][4002].

Reviewer 2: ```suggestion
2. Add your API and App keys to your secrets in Bitrise. Documentation on how to do that can be found here: [Setting a Secret][4002].
```

> 3. Configure your step inputs ([Step Inputs][4003]). You can also configure them in your `bitrise.yml`. The only required inputs are the 2 secrets you configured earlier. The rest of the possible inputs are described in a later section.

Reviewer 2: ```suggestion
3. Configure your step inputs (see the [Step Inputs documentation][4003]). You can also configure them in your `bitrise.yml`. The only required inputs are the two secrets you configured earlier. The rest of the possible inputs are described in a later section.
```

> ## How to use this Step locally

Reviewer 2: ```suggestion
## How to use this step locally
```

Author: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/pull/8#discussion_r1504082222

> This Step can be run directly with the [Bitrise CLI][2003].

Reviewer 2: ```suggestion
This step can be run directly with the [Bitrise CLI][2003].
```

- A_SECRET_PARAM_TWO: the value for secret two
```

## Usage

### Example task using a global configuration override with `configPath`

<!-- TODO: change git urls to step references after we publish it -->
This task overrides the path to the global `datadog-ci.config.json` file.

```yml
- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
   inputs:
   - api_key: $DATADOG_API_KEY
   - app_key: $DATADOG_APP_KEY
   - config_path: './synthetics-config.json'
```

For an example configuration file, see the [`global.config.json` file][2001].

### Example including all possible configurations

> For reference here's how a full configuration could look like:

Reviewer 2: ```suggestion
For reference here's how a full configuration could look:
```
